tools/CMakeLists.txt

```diff
@@ -5,276 +5,276 @@ set(SUBSYS_OPT_DEPS visualization)
 set(DEFAULT ON)
 set(REASON "")
 
-PCL_SUBSYS_OPTION (BUILD_tools "${SUBSYS_NAME}" "${SUBSYS_DESC}" ${DEFAULT} "${REASON}")
-PCL_SUBSYS_DEPEND (BUILD_tools "${SUBSYS_NAME}" DEPS ${SUBSYS_DEPS} OPT_DEPS ${SUBSYS_OPT_DEPS})
+PCL_SUBSYS_OPTION(build "${SUBSYS_NAME}" "${SUBSYS_DESC}" ${DEFAULT} "${REASON}")
```

Comment:
Could you explain this change? Aren't we losing `BUILD_tools` CMake option with this?

Reply:
Thanks, interesting find! An example of how cleaning up and harmonizing style can lead to bug fixes.

Reply:
Ok, this is not a bugfix. This is just an harmonizing. I changed value of `_var` and not of `name`. So before this change value of `BUILD_tools` was changed, but because of missing `PARENT_SCOPE` it has no effect to parent CMakeLists.